src/cryptonote_core/tx_pool.cpp

```diff
@@ -423,8 +436,14 @@ namespace cryptonote
         txpool_tx_meta_t meta;
         if (!m_blockchain.get_txpool_tx_meta(txid, meta))
         {
-          MERROR("Failed to find tx_meta in txpool");
-          return;
+          static bool warned = false;
```

Comment:
Once the cap is reached, it will print here incessantly. I assume it wasn't realized that this would occur because normal usage hasn't breached the default cap to my knowledge.

This matches #6881

https://github.com/monero-project/monero/blob/d52b7d0e99a2135295b4b83deba388844c511b2c/src/cryptonote_core/tx_pool.cpp#L1451-L1459